UI HOVER EFFECT/code/tilt.js

```diff
@@ -0,0 +1,183 @@
+var VanillaTilt = function () {
+    "use strict";
+    class t {
```

Comment:
Please avoid making spaghetti by declaring a class inside a function then instantiating and returning it in an anonymous function. Write clean code by separating all components and write comments such that it clearly shows the flow of your code.